<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="okayButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Okay</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstNameLine">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastNameLine">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressLine">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QLabel" name="mainLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input Player Information</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
